A new beginning: the age of Wayland desktop.

swyrin opened the pull request:
https://github.com/swyrin/dotties/pull/3

- [x] Screenshots
- [x] README
- [x] Xorg -> Wayland


Below is the old log, kept for archiving purposes.
------------------------------------------


Changes made in the repository:

- [ ] Fine-grain the README

- [x] Use HoSF related assets ([Arknights ToS](https://www.arknights.global/terms_of_service))

  > [...]. Yostar Property is not redeemable for any sum of money or monetary value from Yostar or any third party at any time. You further agree not to sell or redeem any Yostar Property for any sum of money or monetary value from any third party. [...] - Section 6

- Rework polybar stuffs
  - [x] Add rainbow r/unixporn text
  - [x] Bring back NumLk and CapLk indicator because I am at my "peak" again
  - [x] Use -git version of polybar on AUR for the rewritten systray feature
  - [ ] Refine the "Now Playing" stuffs since the old one was a CPU hogger
 
- [ ] Rewrite anything.sh (mostly add/delete bloatwares)
  - [ ] An if for literally full installation.
 
- Fix needed keybindings
  - [x] Fix volume change gone oof.
  - [x] Fix slow as f**k brightness change.
  
- [x] Less butthurting live wallpaper

swyrin committed:
Add plank and komorebi

swyrin committed:
Bind flameshot full to PrtScr

swyrin committed:
Remove placeholders for wal

swyrin committed:
Glassware:tm:

swyrin committed:
Redesign bars

swyrin committed:
No longer using wal color for vis

swyrin committed:
Commit my custom dunstrc

swyrin committed:
assets something something

swyrin committed:
Using new rofi setup

swyrin committed:
Add notification on polybar

swyrin committed:
Expand dunst notification width

swyrin committed:
Thunar new stuffs

swyrin committed:
Various changes to configs

swyrin committed:
yoink plank

swyrin committed:
Use native temp module

swyrin committed:
Add more stuffs

- Eww widgets (music viewer, calendar)
- Improved polybar stuffs
- Battery polling script
- Unbloat stuffs

swyrin committed:
Some stuffs for eww?

swyrin committed:
too long @@

- Separate battery check
- Separate polybar config
- Perserve wallpaper settings

swyrin committed:
Welcome Komorebi config

swyrin committed:
Unbloat stuffs

swyrin committed:
Remove hardcoded systemd units

swyrin committed:
Use alias and thinner border

swyrin committed:
Position fix

swyrin committed:
Unbloat sounds

swyrin committed:
Use another GTK theme

swyrin committed:
cleanup polybar config

swyrin committed:
Use file as a wallpaper mode

swyrin committed:
stuffs

swyrin committed:
Fix animated wallpaper keybindings

swyrin committed:
Add more workspaces and more keybind for WM

swyrin committed:
/shrug

swyrin committed:
nuke weather module

swyrin committed:
unbloat pixels

swyrin committed:
P O W E R

swyrin committed:
Add special key and active window module

swyrin committed:
Fix volume control?

swyrin committed:
nuke weather

swyrin committed:
animations, bloatwares and stuffs

swyrin committed:
Use bleeding edge picom

swyrin committed:
Minor change to eww

swyrin committed:
replace dunst

swyrin committed:
A lot of updates, finally

- Mitigate flameshot issue by using latest bspwm
- Use most recent fork of picom for rounded corners + animation + blur
- Add a notification center
- Forgot to install eww
- Use different catpuccin colors
- Use polybar git version for better tray module

swyrin committed:
GTK change theme

swyrin committed:
debloat and tidy up the files

swyrin committed:
add env codes

swyrin committed:
yes

swyrin committed:
use friendlier font config

swyrin committed:
Fix the notification fix

swyrin committed:
bring back the grammar

swyrin committed:
bring back the old color because it is better

swyrin committed:
much changes, wow

- Install firewall with `firewalld`
- Add `make` build optimizations
- Change `nerd-fonts-complete` to `nerd-fonts-git` to match the AUR one
- Add GNOME keyring to comply with Chrome and stuffs
- Move `komorebi`, `pfetch` and `btop` to [Bloatwares] section
- Yeet btop
- Use zen kernel?

swyrin committed:
Bring back animations?

swyrin committed:
Migrate deadd to new config

Swyreee commented:
Stuffs for komorebi will be removed since making it work is pain, will consider finding a new one some day ;)

swyrin committed:
bye bye komorebi

swyrin committed:
debloat picom config

swyrin committed:
forgot to add notification margin

swyrin committed:
I forgot bspwmrc exists

swyrin committed:
update gitignore

swyrin committed:
Removed all files that are in the .gitignore

swyrin committed:
Update anything.sh

swyrin committed:
Change to yellow-ish color

swyrin committed:
Something something new

swyrin committed:
Add shrink keybind

swyrin committed:
oops

swyrin committed:
Bar rename & add pixels

swyrin committed:
Bye bye transparent color

swyrin committed:
new asset

swyrin committed:
Merge remote-tracking branch 'origin/rework' into rework

swyrin committed:
Bring back live wallpaper

with mpv and xwinwrap magic

swyrin committed:
less cursed dunst upgrade

swyrin committed:
improve readability

swyrin committed:
picom?

swyrin committed:
woah, looks glassy

swyrin committed:
fix border color disparity

swyrin committed:
move clipboard manager to a separate keybind

swyrin committed:
styling adjustments

swyrin committed:
new generation of softwares

- PulseAudio to PipeWire
- Separate battery icon and brightness stuff

swyrin committed:
bad picom

swyrin committed:
Add numlock auto starter

swyrin committed:
Reduce update interval of battery icon

swyrin committed:
Remove a useless line

swyrin committed:
Fix wrong keybind

swyrin committed:
a lot of changes right now

- Intro XDG autostart with dex
- Symlink more stuffs
- More systemd (ew bloat) units

swyrin committed:
commit everything before I leave this laptop

swyrin committed:
The age of Wayland. (#3)

swyrin committed:
Merge remote-tracking branch 'origin/main' into rework

swyrin committed:
Upload screenshots.

swyrin committed:
Update README.md
